.github/workflows/Tests.yaml

```diff
@@ -433,3 +429,88 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+
+  arch_datastructures_tests:
+    name: Archs
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler:
+          - gcc-7
+          - gcc-9
+          - clang-6.0
+          - clang-10
+        include:
+          - sde_arch: ("-nhm;nehalem" "-snb;sandybridge" "-hsw;haswell"
+              "-skl;skylake")
+            compiler: gcc-7
+          - sde_arch: ("-nhm;nehalem" "-snb;sandybridge" "-hsw;haswell"
+              "-skl;skylake" "-icx;icelake-server")
+            compiler: gcc-9
+          - sde_arch: ("-nhm;nehalem" "-snb;sandybridge" "-hsw;haswell"
+              "-skl;skylake")
+            compiler: clang-6.0
+          - sde_arch: ("-nhm;nehalem" "-snb;sandybridge" "-hsw;haswell"
+              "-skl;skylake" "-icx;icelake-server" "-tgl;tigerlake")
+            compiler: clang-10
+    container:
+      image: moxcodes/spectrebuildenv:archtests
```

Comment:
This test still uses your container

Reply:
aw, shoot -- the trick of just dropping that commit didn't work with the squashes. fixing now...